Marlin/Marlin_main.cpp

```diff
@@ -782,7 +782,7 @@ extern "C" {
 #endif // !SDSUPPORT
 
 #if ENABLED(DIGIPOT_I2C)
-  extern void digipot_i2c_set_current(int channel, float current);
+  extern void digipot_i2c_set_current(uint8_t channel, float current);
```

Comment:
@thinkyhead speaking of optimizing, couldn't [DAC_STEPPER_CURRENT](https://github.com/MarlinFirmware/Marlin/blob/99d18ecd0cc9be27cfc32f37cc7d1b476b4139a1/Marlin/stepper_dac.cpp#L52) be changed from uint16_t to uint8_t since driver percent is always 0-100 ?